.github/workflows/main.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         - "3.10"
         - "3.11"
         - "3.12"
-        - "3.13-dev"
+        - "3.13.0-alpha.5"
```

Comment:
It doesn't use deadsnakes unless its `-dev`, this is coming from the "core" apt registry

Reply:
I'd like to understand a bit more about the cause before making changes to our CI configuration, and initially would prefer temporarily disabling `python3.13-dev` instead of fixing a specific image name.

Note also that fuzzing is temporarily disabled in Python's continuous integration at the moment (ref python/cpython#116886)

Here's where the `deadsnakes` action code begins to figure out what image to retrieve:

https://github.com/deadsnakes/action/blob/8e851ac3dbb4b8d562383da899c9876e23dcb6b2/bin/install-python#L44-L46

Reply:
> would prefer temporarily disabling python3.13-dev instead of fixing a specific image name

I feel temporarily changing the name is much preferable than entirely disabling it;
at the end if the day `python3.13.0-a5` `python3.13-dev` will be exactly the same for the next month (see https://peps.python.org/pep-0719/)

Reply:
@danieleades @picnixz thoughts? maybe this issue will just fix itself soon, I dunno 🤷

Reply:
Is it possible to allow failures for the 3.13 target?

Reply:
> Is it possible to allow failures for the 3.13 target?

you can already merge with it failing, because its not on the required list, but that does not feel ideal

Reply:
Sorry I won't be available today anymore. In the future, I don't care about the PRs meant to patch the CI/CD since it's only for us and has no downstream effect. So I leave it to you

Reply:
More generally I want don't want to merge to master if tests are failing.

Reply:
The problem seems to relate to HTTP 503 responses from the Ubuntu PPA serving the nightly images:

https://github.com/sphinx-doc/sphinx/actions/runs/8361626870/job/22890154532?pr=12149#step:4:54

> W: Failed to fetch https://ppa.launchpadcontent.net/deadsnakes/nightly/ubuntu/dists/jammy/InRelease  503  Service Unavailable [IP: .... 443]
> W: Failed to fetch https://ppa.launchpadcontent.net/ubuntu-toolchain-r/test/ubuntu/dists/jammy/InRelease  503  Service Unavailable [IP: .... 443]

My guess is that it's related to this infrastructure maintenance: https://ubuntu.social/@launchpadstatus

Reply:
Ubuntu PPA service should now be restored; I'll update one of my pull request branches in a few moments to check.

Reply:
> Ubuntu PPA service should now be restored; I'll update one of my pull request branches in a few moments to check.

:tada: Yep - that's functional again.  Welcome back, `3.13-dev`!  https://github.com/sphinx-doc/sphinx/actions/runs/8364765692/job/22900905325?pr=12144